sdk/tests/conformance/textures/misc/video-rotation.html

```diff
@@ -104,6 +104,9 @@
             gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_S, gl.CLAMP_TO_EDGE);
             gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_T, gl.CLAMP_TO_EDGE);
 
+            // These files were created by converting exif-orientation-test-1-normal.jpg to mp4
+            // files, rotating them using the transpose filter, and adding rotate metadata, all
+            // using the ffmpeg CLI.
```

Comment:
Could you please include the complete command lines you used? If we have to recreate these resources in the future we don't want to have to start from scratch. Thanks.